config/source/cli/cli.go

```diff
@@ -0,0 +1,138 @@
+// Package cli is the Cli component of orb.
```

Comment:
Nit: more idomatic to start package comments with // Package XYZ provides ... 
Also still need to change that on other packages I wrote I think